### PR TITLE
Update to new k8s deploy script

### PR DESCRIPTION
**Pull request by mattgauntseo-sentry**
#skip-changelog

The scripts are almost identical - this is largely a file name change.

### CODE REVIEW

**Review by joshuarli — APPROVED** [review]

**Review by olksdr — APPROVED** [review]
LGTM! Assuming the script is in place and will work once this PR merged. 

@mattgauntseo-sentry  could you, please, add the description to the PR with some small context for the change? It would make reviewing such changes much easier.

**Pull request merged**
